TypeCobol/Compiler/Symbols/Symbol.cs

```diff
@@ -86,42 +135,332 @@ public string Name
         }
 
         /// <summary>
-        /// Type changed event.
+        /// The target AST node if any
+        /// </summary>
+        public Node TargetNode
+        {
+            get;
+            internal set;
+        }
+
+        /// <summary>
+        /// A Typed name is the name followed by a type, by default is the name..
         /// </summary>
-        public event System.EventHandler TypeChanged;
+        public virtual string TypedName => Name;
```

Comment:
What exactly is a "typed name" according to our current implementation ?
+ TypedName and FullTypedDotName are never used (even in branch 1260_DataFlowCFG), can we remove these 2 properties ?

Reply:
This would correspond roughly to DataDefinitionPath when manipulating VariableSymbol coming from typedefs. Since it is not used, i'll remove them.